.github/workflows/release.yml

```
@@ -209,6 +210,23 @@ jobs:
       PYPI_API_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
       TEST_PYPI_API_TOKEN: ${{ secrets.TEST_PYPI_API_TOKEN }}
 
+  docker-core-release:
+    name: Docker dbt-core Release
+    if: ${{ !inputs.test_run }}
+    needs: [pypi-release]
+
+    uses: dbt-labs/dbt-core/.github/workflows/release-docker.yml@main
+    with:
+      package: dbt-core
+
+  docker-postgres-release:
+    name: Docker dbt-postgres Release
+    needs: [docker-core-release]
```

I would explicitly add this for clarity.
```suggestion
  docker-postgres-release:
    name: Docker dbt-postgres Release
    if: ${{ !inputs.test_run }}
    needs: [docker-core-release]
```